docs: release notes for extending private installations (#2716)
commercetools/merchant-center-application-kit: website/src/releases/2022-07/extending-private-installations.mdx

---
date: 2022-07-20
title: Installing Custom Applications in multiple Organizations
description: It is now possible to install a Custom Application in other Organizations where the user has administration rights
type: announcement
topics:
  - Community
  - Merchant Center
---

> With the [release of Org-level Custom Applications](/releases/2022-02-02-custom-applications-v21) one of the main long-term features with this new setup is to be able to configure a Custom Application once and install it _everywhere_.

Reviewer: With the [release of Org-level Custom Applications](/releases/2022-02-02-custom-applications-v21), one of the long-term goals for this feature was to enable Custom Applications to be configured only once and installed _everywhere_ needed.
```suggestion
With the [release of Org-level Custom Applications](/releases/2022-02-02-custom-applications-v21) one of the main long-term features with this new setup is to be able to configure a Custom Application once and install it _everywhere_.
```

> Up until now this functionality was limited to install a Custom Application in the same Organization where it was configured. This limitation was mainly defined to reduce the scope of the initial release. However, enabling Custom Applications to be easily installed in multiple Organizations remains one of the long-term goals for Custom Applications.

Reviewer: Rephrased a bit + 3rd sentence seems to repeat previous paragraph
```suggestion
Up until now this functionality was limited to installing a Custom Application in the same Organization where it was configured. This limitation was mainly posed to reduce the scope of the initial release.
```
